src/lib/contents/authors.ts

```diff
@@ -29,6 +29,14 @@ export const authors: { [idx: string]: Author } = {
     },
     description: "",
   },
+  kpkaiser: {
```

Comment:
```suggestion
  burningion: {
```


I would suggest you use get an avatar (GitHub Profile picture rendered): 


<img width="879" alt="image" src="https://user-images.githubusercontent.com/55068936/217327355-93f2affe-e2b5-473f-a562-69eee79e365c.png">